### DISCUSSION TITLE
[Bug]: Update Pimcore 10.2.8 to 10.3.2

### DISCUSSION THREAD

**Discussion by shaselboeck** [pimcore/pimcore#11580] [Q&A / Support]
### Expected behavior

No errors

### Actual behavior

I performed an update from Pimcore 10.2.8 to 10.3.2. In the Composer Report the following warnings/errors appears:

![composer-report](https://user-images.githubusercontent.com/1867700/156520582-c9e3eb50-edba-4051-9404-bdfe78da1b0b.png)

### Steps to reproduce

Not possible. 10.3.2 is up and running. Not sure, if this warning is really a problem ...?!

Cheers
Sven

**Comment by kingjia90** [comment]
my 2 cents on this, since the problem came from something in `var/cache/prod/Container2a4uv14/App_KernelProdContainer.php`, it should be gone after that successful clear cache. To be sure, i would run cache:clear again to see if the warnings re-appear.
I guess to avoid this, the  `var/cache` needs to be flushed before any update.
